apps/dashboard/src/lib/components/Modal/DeleteModal.svelte

```diff
@@ -32,12 +33,15 @@
         variant={VARIANTS.OUTLINED}
         label={$t('delete_modal.no')}
         onClick={() => (open = false)}
+        isDisabled={isLoading}
```

Comment:
Just use isLoading prop here, it automatically makes the button disabled.

Same for the other function in this file